src/modules/Channel/components/Message/index.tsx

```diff
@@ -49,9 +48,10 @@ const Message = (props: MessageProps): React.ReactElement => {
         !initialized || isDisabledBecauseFrozen(currentGroupChannel) || isDisabledBecauseMuted(currentGroupChannel) || !config.isOnline
       }
       shouldRenderSuggestedReplies={
-        message.messageId === currentGroupChannel?.lastMessage?.messageId
+        config?.groupChannel?.enableSuggestedReplies
```

Comment:
suddently this is undefined now and I can't figure out why. only this is undefined and other configs are set correctly.

Reply:
Would you like to try `yarn install`? If you're working on moving branches, it could be a version issue.

Reply:
yarn install did not fix it but i think its related to moving branches can't figure out why. I tried 
npm cache clean --force
as well but no luck

Reply:
I fixed it by 
1. rm -rf node_modules
2. npm cache clean --force
3. rerun storybook

Reply:
Did you run yarn install and rerun the Storybook?
Because most hot module replacements ([HMR](https://webpack.kr/concepts/hot-module-replacement/)) do not detect changes in node_modules and do not rebuild.
If there is a running environment, they must be rerun.